### ISSUE TITLE
B027 false-positive on `@overload`ed methods

### ISSUE THREAD

**Issue by jwodder** [PyCQA/flake8-bugbear#304]
Running flake8 on the following code with flake8-bugbear 22.10.25:

```python
import abc
from typing import IO, Literal, overload

class Opener(abc.ABC):
    @overload
    def open(self, mode: Literal["r"] = "r") -> IO[str]:
        ...

    @overload
    def open(self, mode: Literal["rb"] = "rb") -> IO[bytes]:
        ...

    @abc.abstractmethod
    def open(self, mode: Literal["r", "rb"] = "r") -> IO:
        ...
```

results in reports of B027 violations for the `@overload`-decorated `open()` methods, which is wrong.

**Comment by jakkdl** [comment]
Yeah `@overload` should definitely be ignored as well. @Zac-HD
